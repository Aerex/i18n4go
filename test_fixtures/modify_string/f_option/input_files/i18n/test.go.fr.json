[
   {
      "id": "Hello modify string",
      "translation": "Hello modify string"
   }
]