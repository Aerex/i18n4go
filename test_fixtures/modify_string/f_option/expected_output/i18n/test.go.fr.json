[
   {
      "id": "Hello modify string command",
      "translation": "Hello modify string"
   }
]